src/Tools/Console/Command/ReservedWordsCommand.php

```diff
@@ -113,11 +110,9 @@ protected function configure(): void
     /**
      * {@inheritdoc}
      *
-     * @return int
-     *
      * @throws Exception
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
This change looks irrelevant.

Reply:
I'd prefer "unrelated", but you're right, I'll revert it. 🙂

Reply:
Moved to #5074

Reply:
I stand corrected.